shuttle/index.php

```diff
@@ -0,0 +1,45 @@
+<?php
+use ThauEx\SimpleHtmlDom\SHD;
+
+SHD::$fileCacheDir = "cache";
+$html = SHD::fileGetHtml('http://wzw.tum.de/index.php?id=416');
```

Comment:
The String can be extracted as a constant or global variable, as it is used several times